src/include/OpenImageIO/typedesc.h

```diff
@@ -85,7 +85,7 @@ struct OIIO_API TypeDesc {
                     HALF, FLOAT, DOUBLE, STRING, PTR, LASTBASE };
     /// AGGREGATE describes whether our type is a simple scalar of
     /// one of the BASETYPE's, or one of several simple aggregates.
-    enum AGGREGATE { SCALAR=1, VEC2=2, VEC3=3, VEC4=4, MATRIX44=16 };
+    enum AGGREGATE { SCALAR=1, VEC2=2, VEC3=3, VEC4=4, MATRIX33=9, MATRIX44=16 };
```

Comment:
Hopefully. :-)

I mean, we could add another tag for MATRIX22, but it would break the current scheme of having the AGGREGATE enum's numeric representation be the number of scalars, it makes the computation of the total size of the type really simple without extra conditionals or table lookups.

There were many possible ways to organize TypeDesc's internal layout. At some point it was an arbitrary choice. I'm digressing, but another one I sometimes wish I'd done would be to separate the base type from one enum into orthogonal "int or float", "signed or unsigned", and "number of bits".